c.html.markdown

```diff
@@ -236,11 +236,9 @@ int main (int argc, char** argv)
   z = (e > f) ? e : f; // => 10 "if e > f return e, else return f."
 
   // Increment and decrement operators:
-  char *s = "ILoveC";
   int j = 0;
-  s[j++]; // => "I". Returns the j-th item of s THEN increments value of j.
-  j = 0;
-  s[++j]; // => "L". Increments value of j THEN returns j-th value of s.
+  int s = j++; // Return j THEN increase j. (s -> 0, j -> 1)
+  s = ++j; // Increase j THEN return j. (s -> 2, j -> 2)
```

Comment:
@vendethiel 

No special reason.
`a -> b` reads like `a is changed to b` to me.
Have not thought that A `->` is typically used in function `parameter -> result`.
I will push a commit fix this. Thanks.